src/test/java/teammates/storage/api/NotificationsDbTest.java

```diff
@@ -191,12 +191,12 @@ public void testUpdateNotification_singleFieldUpdate_shouldUpdateSuccessfully()
                 notificationsDb.getNotification(n.getNotificationId()).getMessage());
 
         ______TS("success: single field - type");
```

Comment:
```suggestion
        ______TS("success: single field - style");
```

Reply:
Thank you very much. I will change together after @t-cheepeng takes a look at this PR.